Example/Backpack/BPKExampleApp.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+@import UIKit;
```

Comment:
Can we stick to `#import <UIKit/UIKit.h>` for the time being? We might conclude that we should use the `@import` syntax more, but let's make that an explicit decision if so

Reply:
I didn't even know about this syntax!

Reply:
It was introduce a while back and is specific to frameworks(I think?). See https://stackoverflow.com/questions/18947516/import-vs-import-ios-7